docs/testing/platform-limits.md

```diff
@@ -44,7 +44,7 @@ If you're encountering errors when dealing with large datasets, try using multip
 
 ### Power Automate
 
-When using Office Scripts with Power Automate, each user is limited to **800 calls to the Run Script action per day**. This limit resets at 12:00 AM UTC.
+When using Office Scripts with Power Automate, each user is limited to **1600 calls to the Run Script action per day**. This limit resets at 12:00 AM UTC.
```

Comment:
Actually, the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33688) says to use commas for large numbers. I'll make the edit.
```suggestion
When using Office Scripts with Power Automate, each user is limited to **1,600 calls to the Run Script action per day**. This limit resets at 12:00 AM UTC.
```